aws-replicator/aws_replicator/client/auth_proxy.py

```diff
@@ -55,9 +55,9 @@
     LOG.setLevel(logging.DEBUG)
 
 # TODO make configurable
-CLI_PIP_PACKAGE = "localstack-extension-aws-replicator"
+# CLI_PIP_PACKAGE = "localstack-extension-aws-replicator"
 # note: enable the line below temporarily for testing:
-# CLI_PIP_PACKAGE = "git+https://github.com/localstack/localstack-extensions/@branch#egg=localstack-extension-aws-replicator&subdirectory=aws-replicator"
+CLI_PIP_PACKAGE = "git+https://github.com/localstack/localstack-extensions/@fix/aws-proxy-extension#egg=localstack-extension-aws-replicator&subdirectory=aws-replicator"
```

Comment:
Attempting to run this from this branch directly doesn't work because it also needs this line to be set to the branch, so in order for Audacy's issue to be fixed, we need a release for this.